src/index.html

```diff
@@ -1,13 +1,39 @@
 <!DOCTYPE html>
 <html lang="en">
```

Comment:
- [ ] **Required:** You have an amazing setup in which you used [GitHub flow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/github_flow.md). However, this project required [Git flow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/gitflow.md). Your PR should be from a `dev`, `develop` or `development` branch.
- You should make the develop branch the default and then make all merges to it as [illustrated here.](https://youtu.be/d0WEe8xH5mk)

- [ ] **Required** Practice using the ES6 arrow function rather than the function keyword as this module is designed to give you the necessary knowledge on the use of ES6 syntax. https://github.com/Tobinchilongo/Leaderboard/blob/2a1a7246013965993757d4d9c09de8421d1d6e4f/src/index.js#L4